src/core/components/Dropzone/Dropzone.tsx

```diff
@@ -81,7 +87,14 @@ const Dropzone = (props: DropzoneProps) => {
       {fileRejections.length + acceptedFiles.length === 0 && children}
       {acceptedFiles.length > 0 && (
         <span className="line-clamp-3 text-xs">
-          {acceptedFiles?.map((f) => f.name).join(", ")}
+          <ul>
```

Comment:
Small welcome UI change :)

Reply:
Thanks. As we are still limiting it to 3 displayed lines, having the number of files selected would be a good addition as well but it's a mere bonus